interoperator/cmd/manager/main.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
none of our other src files have any license info header. any reasons to add this here?

Reply:
I am pretty sure it is by the kubebuilder tool, as it needs a licence file to initialize. May be it adds it only in the main.go file, which is the startup file.